system/mp/test/mptest.cpp

```diff
@@ -9,16 +9,19 @@
 #include <jcrc.hpp>
 #include <mpbase.hpp>
 #include <mpcomm.hpp>
+#include "mplog.hpp"
```

Comment:
mplog is something special to do with redirection of logs between components on different machines..
we shouldn't be extending or using it directly (when it's used it's used automatically via hooks)